### PR TITLE
Add navigations

### PR DESCRIPTION
**Pull request by mengX666**
Add `AtlassianNavigation` and `SideNavigation` to the homepage.
demo: 
image

**How to view the changes:**
in `app.tsx`:
```
import Home from "./home";

// in return function

  return <Home />
```

### CODE REVIEW

File: template/src/home/sample-logo.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
+const RocketIcon = () => {
```
Comment (abruzzi) [inline template/src/home/sample-logo.tsx:1]: I like the way you import these icons other than bring the whole `atlaskit/icon` package in.

File: template/src/home/side-navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import WorkIcon from "@atlaskit/icon/glyph/folder";
+import CustomerIcon from "@atlaskit/icon/glyph/person";
```
Comment (abruzzi) [inline template/src/home/side-navigation.tsx:2]: hmm, what's your preference of importing an icon versus copy one into the codebase?

Comment (mengX666) [reply template/src/home/side-navigation.tsx:2]: haven't really thought about this 🤔  `atlaskit/icon` is the dependency of `form` and `atlassian-navigation` anyway so I just import these icons. What are the pros and cons of each?

Comment (abruzzi) [reply template/src/home/side-navigation.tsx]: Hi @mengX666 that's an interesting one, I guess the longer you stay in lite-mode, the sensitive you will be - so any direct import looks suspicious to me. Seems it doesn't really matter if the icon is installed with form and navigation anyways.

**Comment by abruzzi** [comment]
Hi @mengX666 I added `react-router-dom` to connect the two pages (and did a little bit restructure of the pages). Please have a look when you're free.

Cheers. 🥂

File: template/src/home/index.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        style={{
+          height: `calc(100% - 56px)`,
+          minHeight: 600,
+          display: "flex",
+        }}
+      >
+        <div
+          style={{
+            minHeight: 600,
+            borderRight: "1px solid #DFE1E6",
```
Comment (mengX666) [inline template/src/home/index.tsx:41]: Just curious any reason for using hex value instead of the colour variable in atlassian/theme here?

**Comment by mengX666** [comment]
Not relating to this PR, but found this behaviour is probably not expected 🤔
Mar-28-2021 20-52-10

File: template/src/home/side-navigation.tsx (hunk 2 above)
Comment (mengX666) [reply template/src/home/side-navigation.tsx]: Keen to know more about this, could we schedule a quick call tmrw? 🙏

**Comment by abruzzi** [comment]
> Not relating to this PR, but found this behaviour is probably not expected 🤔
> Mar-28-2021 20-52-10

I think it may be expected - maybe final-form enlarged the clickable area for the field?

File: template/src/home/side-navigation.tsx (hunk 2 above)
Comment (abruzzi) [reply template/src/home/side-navigation.tsx]: > Keen to know more about this, could we schedule a quick call tmrw? 🙏

sure, let's do that some time tomorrow.

**Pull request merged**